concurrency-limits-core/src/main/java/com/netflix/concurrency/limits/limit/WindowedLimit.java

```diff
@@ -74,6 +80,17 @@ public Builder minRttThreshold(long threshold, TimeUnit units) {
             return this;
         }
 
+        public Builder percentile() {
+            return percentile(DEFAULT_PERCENTILE);
+        }
+
+        public Builder percentile(int percentile) {
+            Preconditions.checkArgument(percentile >= 0 && percentile <= 100, "Percentile should belong to [0, 100]");
```

Comment:
Let's make percentile a double in the range (0, 1.0) instead of using ints.

Reply:
> (0, 1.0)

By writing that did you mean mathematical notation, therefore inducing that `0` and `1.0` should not be allowed here?

Reply:
Correct, we could be more restrictive but this feels like a good start.  Maybe we want to limit the minimum to 50th percentile and max to 99.5th.  Not sure it's worth taking an opinion on that though.